cli/src/main/java/com/devonfw/tools/ide/tool/ToolCommandlet.java

```diff
@@ -93,7 +93,7 @@ public void run() {
    *        without touching and IDE installation and used to run.
    * @param args the commandline arguments to run the tool.
    */
-  public void runTool(VersionIdentifier toolVersion, String... args) {
+  public void runTool(boolean isBackgroundProcess, VersionIdentifier toolVersion, String... args) {
```

Comment:
Please adjust javadoc, isBackgroundProcess is missing here.

Reply:
Done